caexporter.go

```diff
@@ -34,6 +34,11 @@ var (
 		[]string{"activity"},
 		nil,
 	)
+
+	scrape_error = prometheus.NewGauge(prometheus.GaugeOpts{
```

Comment:
The variable names in go usually ar `camelCase` not `snake_case`. So suggest to tename the variable to `scrapeError`. Also as mentioned in previous CR, set a namespace for the metric.